.github/workflows/check-i18n-changes.yml

```
@@ -0,0 +1,23 @@
+name: Protect content from manual changes
+
+on: 
+  pull_request:
+    paths:
+    - 'content/**'
+
+jobs:
+  creator-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+
+      - name: 
+        if: github.actor != "nodejs-crowdin" || github.actor != "github-actions"
```

`github.actor = 'github-actions'` will returns `true`. The the operator end here
```suggestion
        if: github.actor != "nodejs-crowdin" && github.actor != "github-actions"
```